src/app/models/propTypes/article/index.js

```diff
@@ -0,0 +1,50 @@
+import { array, arrayOf, number, shape, string } from 'prop-types';
+
+const articlePropTypes = {
+  data: shape({
+    metadata: shape({
+      id: string.isRequired,
+      type: string.isRequired,
+      createdBy: string,
+      blockId: string.isRequired,
+      created: number.isRequired,
+      firstPublished: number.isRequired,
+      lastPublished: number.isRequired,
+      lastUpdated: number.isRequired,
+      locators: shape({
+        optimoUrn: string.isRequired,
+      }),
+      passport: shape({
+        language: string.isRequired,
+        home: string.isRequired,
+        articleType: string.isRequired,
+        genre: string,
+      }),
+      tags: {
+        about: string,
+        mentions: string,
+      },
+      version: string,
+      blockTypes: arrayOf(string),
+    }).isRequired,
+    content: shape({
+      model: shape({
+        blocks: array,
```

Comment:
We should use this https://github.com/bbc/simorgh/blob/latest/src/app/models/propTypes/mainContent/index.js#L6.